pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -148,6 +148,12 @@ type AlertmanagerSpec struct {
 	// and the actual ExternalURL is still true, but the server serves requests
 	// under a different route prefix. For example for use with `kubectl proxy`.
 	RoutePrefix string `json:"routePrefix,omitempty"`
+	// Maximum number of GET requests processed concurrently.
```

Comment:
Same as above .. I did think of copying the text from Alertmanager but I think we shouldn't rely on that since the behaviour may change in future. Rather the statement should be generic and specific behaviour depends on the Alertmanager version. .. what say?